src/file.cpp

```diff
@@ -80,18 +80,18 @@ namespace Sass {
     bool file_exists(const std::string& path)
     {
       #ifdef _WIN32
-        wchar_t resolved[32768];
+        std::vector<wchar_t> resolved(32768);
         // windows unicode filepaths are encoded in utf16
         std::string abspath(join_paths(get_cwd(), path));
         if (!(abspath[0] == '/' && abspath[1] == '/')) {
           abspath = "//?/" + abspath;
         }
         std::wstring wpath(UTF_8::convert_to_utf16(abspath));
         std::replace(wpath.begin(), wpath.end(), '/', '\\');
-        DWORD rv = GetFullPathNameW(wpath.c_str(), 32767, resolved, NULL);
+        DWORD rv = GetFullPathNameW(wpath.c_str(), 32767, &resolved[0], NULL);
```

Comment:
```suggestion
        DWORD rv = GetFullPathNameW(wpath.c_str(), max_chars, resolved.data(), NULL);
```

Reply:
This shouldn't be necessary, since GetFullPathNameW will fill the buffer anyway!

Reply:
This suggestion is to avoid repeating `32767` and to use `.data()` instead of `&[0]`